package.json

```diff
@@ -10,15 +10,13 @@
     "lint": "tslint ./packages/**/src/**/*.ts --exclude \"./packages/**/node_modules/**/*.ts\"",
     "prebootstrap": "npm install",
     "postinstall": "lerna bootstrap",
-    "pretest": "npm run compile",
```

Comment:
Compilation for the tests and code coverage is done on the fly. We can use ts-node to compile typescript files when we require them. This removes the need to precompile code to run tests